druid/examples/image.rs

```diff
@@ -12,44 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! This example shows how to draw an png image.
+//! This example shows how to draw an image. Using the Image feature.
+//! Diferent image formats can be "unlocked" by adding their
+//! corresponding feature.
 
-use druid::{
-    widget::{FillStrat, Flex, Image, WidgetExt},
-    AppLauncher, Color, ImageBuf, Widget, WindowDesc,
-};
+use druid::piet::InterpolationMode;
+use druid::widget::prelude::*;
+use druid::widget::{FillStrat, Flex, Image, WidgetExt};
+use druid::{AppLauncher, Color, ImageBuf, WindowDesc};
 
 pub fn main() {
     let main_window = WindowDesc::new(ui_builder);
-    let data = 0_u32;
```

Comment:
although it might seem useless, the idea here is to pretend more like we have actual app data, and also to make it clear that it is the app data that is passed in with launch. I don't think it matters much either way, but I want to at least explain the thinking. :)